### PR TITLE
[4.0] Global configuration options add media-breakpoint-down(sm)

### PR DESCRIPTION
**Pull request by astridx**
Pull Request for Issue https://github.com/joomla/joomla-cms/pull/27613#issuecomment-577838109 .

### Summary of Changes
I added a `media-breakpoint-down(sm)` for the options. On small devices the label ob an option is above this option.

### Testing Instructions

1.   Open Global Configuration an see that all options displayed beside the label.
2.   Apply this patch and run `npm ci`
3.   Open Global Configuration again and see, that the options on small devices is displayed in there own row above the label.

### Expected result
Global Configuration   test   Administration(2)

### Actual result
Global Configuration   test   Administration(3)

### Documentation Changes Required
No

### CODE REVIEW

**Comment by richard67** [comment]
@astridx When testing this PR here, shall we also apply PR #27613 ? If yes, then please mention that in testing instructions.

**Comment by astridx** [comment]
@richard67 It is not necessary to test both PRs together. Both can be tested independently.

**Comment by richard67** [comment]
@astridx Thanks for clarification. Was not sure about it because your 2nd screenshot looks to me as if you had applied both.

**Comment by astridx** [comment]
@richard67 No, I hadn't applied both PRs together. A column was previously set on small displays.

**Comment by richard67** [comment]
@astridx Yes, I just see while testing ;-)

**Comment by astridx** [comment]
@richard67 Thank you very much for testing :)

**Comment by richard67** [comment]
@astridx Please add the information to your testing instructions to run "npm install" or "npm i" or "npm run build:css" after having applied the patch, if not using patchtester 4 (if that works for someone).

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 5fa1555b21dfdd26519ec06a24c863a8f92dfb65<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27615">issues.joomla.org/tracker/joomla-cms/27615</a>.</sub>

**Comment by Quy** [comment]
Missing label on switcher. The missing value is fixed in #27098.

**Comment by richard67** [comment]
Confirmed.

**Comment by richard67** [comment]
I have not tested this item.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27615">issues.joomla.org/tracker/joomla-cms/27615</a>.</sub>

**Comment by richard67** [comment]
Changed back my test result to "Not tested" due to issue mentioned above by @Quy .

**Comment by astridx** [comment]
Oops, I didn't notice that either.

**Comment by richard67** [comment]
@astridx Don't forget to update your testing instructions regarding usage of npm.

**Comment by astridx** [comment]
@richard67 I added the need to run npm ci in the test descriptions.

**Comment by astridx** [comment]
The label for the switch was not shown because the surrounding `div` had the class `sr-only`. I have now changed this. I see no reason why this should be screenreader-only `sr-only`.

**Comment by Quy** [comment]
Switcher should be under label.

27615b

**Comment by infograf768** [comment]
Issue with switcher labels is due to the presence of a float for `.control-group .control-label`
plus we have 2 instances of the label, one as label the other as legend

File: composer.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
-    "name": "joomla/joomla-cms",
+    "name": "astridx/joomla-cms",
     "type": "project",
     "description": "Joomla CMS",
     "keywords": ["joomla", "cms"],
-    "homepage": "https://github.com/joomla/joomla-cms",
+    "homepage": "https://github.com/astridx/joomla-cms",
```
Comment (alikon) [inline composer.json]: @astridx `https://github.com/astridx/joomla-cms`  should be always joomla :smile:

Comment (richard67) [reply composer.json]: @alikon Are you sure? Maybe Joomla has been renamed again? (joke)

**Comment by astridx** [comment]
@infograf768 Thank for pointing me in the right direction.

Please have a look to see if it fits now. 

I intentionally made the changes in file /administrator/templates/atum/scss/blocks/_form.scss.
I think that the file /build/media_source/system/scss/fields/switcher.scss should remain independent of the number of columns. 

Do you also see it like that?

File: composer.json (hunk 1 above)
Comment (astridx) [reply composer.json]: I noticed that. That should have already been corrected. Sorry for the mess

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 046a1b1f67ca542f8f76cdf83d9f7adc35ea0240<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27615">issues.joomla.org/tracker/joomla-cms/27615</a>.</sub>

File: administrator/templates/atum/scss/blocks/_form.scss [hunk 2]
```diff
@@ -37,6 +46,20 @@
   }
 }
 
+.col-md-9 .control-group .switcher__legend,
+.col-md-12 .control-group .switcher__legend {
+
+    @include media-breakpoint-down(sm)
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_form.scss]: Indentation should be 2 spaces. Fix up to line 58.

Comment (astridx) [reply administrator/templates/atum/scss/blocks/_form.scss]: Corrected the indentation

File: administrator/templates/atum/scss/blocks/_form.scss [hunk 3]
```diff
@@ -28,6 +28,15 @@
 
   .controls {
     margin-left: 220px;
+
+    @include media-breakpoint-down(sm)
+    {
```
Comment (richard67) [inline administrator/templates/atum/scss/blocks/_form.scss]: Why here a new line for the opening curly bracket, and elsewhere no new line? Just asking, not sure if right or wrong.

Comment (astridx) [reply administrator/templates/atum/scss/blocks/_form.scss]: The curly bracket is now like everywhere.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 198746d84ccea75cdb388fbd11b0b87be969c3ae<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27615">issues.joomla.org/tracker/joomla-cms/27615</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 198746d84ccea75cdb388fbd11b0b87be969c3ae<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27615">issues.joomla.org/tracker/joomla-cms/27615</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27615">issues.joomla.org/tracker/joomla-cms/27615</a>.</sub>

**Comment by Quy** [comment]
Thanks! Can you also please look into the Permissions tab in a separate PR?

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**